integration-tests/src/test/resources/setupenv.sh

```diff
@@ -87,7 +87,7 @@ function pull_tag_images {
 	  echo "Creating Docker Secret"
 	  
 	  kubectl create secret docker-registry $IMAGE_PULL_SECRET_WEBLOGIC  \
-	    --docker-server=index.docker.io/v1/ \
+	    --docker-server=container-registry.oracle.com/ \
```

Comment:
Add a variable WL_DOCKER_SERVER and use that in the script

Reply:
Done